hello-world.ks.in

```diff
@@ -0,0 +1,44 @@
+#test name: hello-world
+
+%ksappend common/common.ks
+%ksappend repos/default.ks
+%ksappend l10n/default.ks
+%ksappend users/default.ks
+%ksappend network/default.ks
+%ksappend storage/default.ks
+%ksappend payload/default_packages.ks
+
+%addon org_fedora_hello_world --reverse
+It is raining cats and dogs!
+%end
+
+%pre
+# The URL of the Hello Word repository
+SOURCE="https://github.com/rhinstaller/hello-world-anaconda-addon/archive/master.tar.gz"
```

Comment:
I just realized that this won't work for anything else except Rawhide. I guess this should be fragment or variable. I would prefer a fragment.